tests/test_verify_upgrade_for_remaining_v1.py

```diff
@@ -0,0 +1,225 @@
+import brownie
+from brownie import interface, SettV1h, ERC20Upgradeable, accounts
+import pytest
+from badger_utils.token_utils.distribute_from_whales_realtime import (
+    distribute_from_whales_realtime_percentage
+)
+
+"""
+Tests for Upgrading Sett V1 to SettV1h
+"""
+
+# Remaining Setts
+
+# bBADGER
+# https://etherscan.io/address/0x19D97D8fA813EE2f51aD4B4e04EA08bAf4DFfC28
+# SettV1
+
+# bcrvTBTC
+# https://etherscan.io/address/0xb9D076fDe463dbc9f915E5392F807315Bf940334
+# SettV1
+
+# bharvestcrvRenBTC
+# https://etherscan.io/address/0xAf5A1DECfa95BAF63E0084a35c62592B774A2A87
+# SettV1
+
+# buniWbtcBadger
+# https://etherscan.io/address/0x235c9e24D3FB2FAFd58a2E49D454Fdcd2DBf7FF1
+# SettV1
+
+LIST_OF_EXPLOITERS = [
+    "0xa33B95ea28542Ada32117B60E4F5B4cB7D1Fc19B",
+    "0x4fbf7701b3078B5bed6F3e64dF3AE09650eE7DE5",
+    "0x1B1b391D1026A4e3fB7F082ede068B25358a61F2",
+    "0xEcD91D07b1b6B81d24F2a469de8e47E3fe3050fd",
+    "0x691dA2826AC32BBF2a4b5d6f2A07CE07552A9A8E",
+    "0x91d65D67FC573605bCb0b5E39F9ef6E18aFA1586",
+    "0x0B88A083dc7b8aC2A84eBA02E4acb2e5f2d3063C",
+    "0x2eF1b70F195fd0432f9C36fB2eF7C99629B0398c",
+    "0xbbfD8041EbDE22A7f3e19600B4bab4925Cc97f7D",
+    "0xe06eD65924dB2e7b4c83E07079A424C8a36701E5"
+]
+
+SETT_REMAINING = [
+    "0x19D97D8fA813EE2f51aD4B4e04EA08bAf4DFfC28",
+    "0xb9D076fDe463dbc9f915E5392F807315Bf940334",
+    "0xAf5A1DECfa95BAF63E0084a35c62592B774A2A87",
+    "0x235c9e24D3FB2FAFd58a2E49D454Fdcd2DBf7FF1",
+]
+
+@pytest.mark.parametrize(
+    "settAddress",
+    SETT_REMAINING,
+)
+def test_upgrade_and_harvest(settAddress, proxy_admin, proxy_admin_gov, bve_cvx, bcvx_crv):
+    vault_proxy = SettV1h.at(settAddress)
+
+    if(bve_cvx.paused() and not settAddress == bve_cvx.address):
+        bve_gov = accounts.at(bve_cvx.governance(), force=True)
+        bve_cvx.unpause({"from": bve_gov})
+    
+    if(bcvx_crv.paused() and not settAddress == bcvx_crv.address):
+        bcvx_crv_gov = accounts.at(bcvx_crv.governance(), force=True)
+        bcvx_crv.unpause({"from": bcvx_crv_gov})
+
+    prev_gov = vault_proxy.governance()
+
+    governance = accounts.at(prev_gov, force=True)
+    ## TODO: Add new code that will revert as it's not there yet
+    with brownie.reverts():
+        vault_proxy.patchBalances({"from": governance}) ## Not yet implemented
+    with brownie.reverts():
+        vault_proxy.MULTISIG() ## Not yet implemented
+
+    ## Setting all variables, we'll use them later
+    prev_gov = vault_proxy.governance()
+    prev_keeper = vault_proxy.keeper()
+    prev_token = vault_proxy.token()
+    prev_controller = vault_proxy.controller()
+    prev_balance = vault_proxy.balance()
+    prev_min = vault_proxy.min()
+    prev_max = vault_proxy.max()
+    prev_getPricePerFullShare = vault_proxy.getPricePerFullShare()
+    prev_available = vault_proxy.available()
+
+
+    ## TODO: Add write operations
+    new_vault_logic = SettV1h.deploy({"from": governance})
```

Comment:
Possibly better to use the mainnet deployed logic here instead of redeploying locally

Reply:
Good call, change made. Thanks!